pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.Designer.cs

```diff
@@ -515,6 +518,7 @@ private void InitializeComponent()
             // 
             // contextMenuSpectrum
             // 
+            this.contextMenuSpectrum.ImageScalingSize = new System.Drawing.Size(32, 32);

```

Comment:
This is not allowed and should be failing code inspection. Indicates editing not at 100%